src/nvim/window.c

```diff
@@ -5048,27 +5067,22 @@ int min_rows(void)
   return total;
 }
 
-/*
- * Return TRUE if there is only one window (in the current tab page), not
- * counting a help or preview window, unless it is the current window.
- * Does not count "aucmd_win".
- */
-int only_one_window(void)
+/// Return true if there is only one window (in the current tab page), not
```

Comment:
hmm the () seems to contradict the first `if`. Should be `only one window (and only one tab page)` ?